src/firewall/state.rs

```diff
@@ -272,8 +272,9 @@ mod tests {
             network_hash_name: "hash".to_string(),
             isolation: IsolateOption::Never,
             dns_port: 53,
+            outbound_addr: None,
         };
-        let net_conf_json = r#"{"subnets":["10.0.0.0/24"],"bridge_name":"bridge","network_id":"c2c8a073252874648259997d53b0a1bffa491e21f04bc1bf8609266359931395","network_hash_name":"hash","isolation":"Never","dns_port":53}"#;
+        let net_conf_json = r#"{"subnets":["10.0.0.0/24"],"bridge_name":"bridge","network_id":"c2c8a073252874648259997d53b0a1bffa491e21f04bc1bf8609266359931395","network_hash_name":"hash","isolation":"Never","dns_port":53,"outbound_addr":null}"#;
```

Comment:
This is a breaking change, you break the deserialize from older version as this json is stored on the fs and if netavark is updated in the meantime the firewall reload service will fail to parse that from older versions.

So this change must be reverted and you should instead and a new test case with a outbound_addr defined so we know both versions pass.